sources/Interop/D3D12MemoryAllocator/src/ALLOCATOR_DESC.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+namespace TerraFX.Interop
+{
+    /// <summary>Parameters of created Allocator object. To be used with CreateAllocator().</summary>
+    public unsafe struct ALLOCATOR_DESC
+    {
+        /// <summary>Flags</summary>
+        public ALLOCATOR_FLAGS Flags;
+
+        /// <summary>
+        /// Direct3D device object that the allocator should be attached to.
+        /// <para>Allocator is doing `AddRef`/`Release` on this object.</para>
+        /// </summary>
+        public ID3D12Device* pDevice;
+
+        /// <summary>
+        /// Preferred size of a single `ID3D12Heap` block to be allocated.
+        /// <para>Set to 0 to use default, which is currently 256 MiB.</para>
+        /// </summary>
+        [NativeTypeName("UINT64")] public ulong PreferredBlockSize;
+
+        /// <summary>
+        /// Custom CPU memory allocation callbacks. Optional.
+        /// <para>Optional, can be null. When specified, will be used for all CPU-side memory allocations.</para>
+        /// </summary>
+        public ALLOCATION_CALLBACKS* pAllocationCallbacks;
```

Comment:
nit: C++ is `const` and so this should be:
```suggestion
        [NativeTypeName("const ALLOCATION_CALLBACKS*")]
        public ALLOCATION_CALLBACKS* pAllocationCallbacks;
```

Reply:
Done in 691bd3f0989e49167630255074e59dabfb89a56a for all the public APIs.